protocol/mining/src/rule_engine.rs

```diff
@@ -75,6 +75,8 @@ impl MiningRuleEngine {
                     has_sufficient_peer_connectivity: self.has_sufficient_peer_connectivity(),
                     finality_duration: self.config.finality_duration_in_milliseconds().get(sink_daa_timestamp.daa_score),
                     elapsed_time,
+                    sink_daa_score_timestamp: session.async_get_sink_daa_score_timestamp().await,
```

Comment:
you already have `sink_daa_timestamp` from above (no need to repeat the consensus call)